GroupIS1024/Models/Coach.cs

```diff
@@ -1,23 +1,29 @@
-﻿namespace GroupIS1024.Models
+﻿using Newtonsoft.Json;
+
+namespace GroupIS1024.Models
 {
     public class Coach
     {
+        [JsonProperty("Coach Id")]
         public int CoachId { get; set; }
 
+        [JsonProperty("First Name")]
         public string FirstName { get; set; }
 
+        [JsonProperty("Last Name")]
         public string LastName { get; set; }
 
-        public string Username { get; set; }
+        [JsonProperty("User Name")]
```

Comment:
I am unable to infer the use of Username. I am curious to know more about Username. 

I have changed the variable name case to match the PascalCase practice followed in all other models.

Reply:
Good.